makefile

```diff
@@ -22,33 +22,30 @@ CFLAGS   = $(INCS) -m64 -Wall -pedantic -g3
 RM       = rm -f
 
 ENABLE_OPENCL ?= 1
+ENABLE_COMPLEX_X2 ?= 1
+ENABLE_COMPLEX8 ?= 0
 OPENCL_AMDSDK = /opt/AMDAPPSDK-3.0
 
 ifeq (${ENABLE_OPENCL},1)
   LIBS += -lOpenCL
-  CXXFLAGS += -DENABLE_OPENCL=1 -I$(OPENCL_AMDSDK)/include
+  CXXFLAGS += -I$(OPENCL_AMDSDK)/include
   # Support the AMD SDK OpenCL stack
   ifneq ($(wildcard $(OPENCL_AMDSDK)/.),)
     LDFLAGS += -L$(OPENCL_AMDSDK)/lib/x86_64
   endif
-else
-  CXXFLAGS += -DENABLE_OPENCL=0
 endif
 
 
 .PHONY: all all-before all-after clean clean-custom qrack
 
 all: all-before $(BIN) bin2hex all-after
 
-$(QRACKLIBS):
-	ENABLE_OPENCL=$(ENABLE_OPENCL) ${MAKE} -C qrack/build
```

Comment:
Why did you remove this?

Reply:
Because I was building from installed Qrack rather than local Qrack, for a minute. Reverted.